scripts/select_appointments.sql

```diff
@@ -1 +1,18 @@
 -- retrieve appointment information
+SELECT p.given_name, p.surname, a.time, a.date_of_appointment
+FROM patients p
+INNER JOIN appointments a
+  ON a.patient_id = p.id
+INNER JOIN doctors d
+  ON a.doctor_id = d.id
+WHERE a.doctor_id = 6
+;
+
+SELECT d.surname doctor, a.time, a.date_of_appointment
+FROM patients p
+INNER JOIN appointments a
+  ON a.patient_id = p.id
+INNER JOIN doctors d
```

Comment:
You have the order of the tables reversed here. If you want to pull out the surname of the doctor, you need to pull it out of `d`, not `p`.
```
FROM doctors d
INNER JOIN appointments a
...
INNER JOIN patients p
...
```